src/main/webapp/index.html

```diff
@@ -0,0 +1,80 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <title>Black Student Insight</title>
+    <meta charset="UTF-8">
+    <link rel="stylesheet" href="style.css">
+    <script type="text/javascript" src="https://www.gstatic.com/charts/loader.js"></script>
+    <script src="script.js"></script>
+  </head>
+  <body onload="hideElements()">
+    <header>
+      <a href="index.html"><h1>Black Student Insight</h1></a>
+      <input type="text" id="school-search" placeholder="Find your institution"/>
+      <button onclick="loadSchoolInfo()">Search</button>
+    </header>
+    <div id="information">
+      <h2>Our Focus</h2>
+      <p>Black Student Insight serves as a platform for current Black college students
+        to share their experiences and beliefs regarding their campuses, while also
+        providing prospective students with general information about the schools,
+        such as overall student diversity demographics, admissions statistics, and
+        tuition for their university of interest.
+      </p>
+
+      <h2>Meet The Creators</h2>
+      <div id="headshots">
+        <figure class="inline-block">
+          <a href="https://www.linkedin.com/in/jalen-jenkins-b75772186"><img src="images/jalen.png"
+          alt="Jalen Jenkins" height="200"></a>
+          <figcaption>Jalen is currently a Computer Science student in Cornell
```

Comment:
Similar comment here to my `p` comment above - leave `figcaption` and the closing tag on their own lines and start the child text on a new line, indented by two spaces.